realm/realm-library/src/syncIntegrationTest/java/io/realm/PermissionManagerTests.java

```diff
@@ -506,7 +506,7 @@ public void permissionManagerAsyncTask_handlePermissionRealmError() throws NoSuc
         // Simulate error in the permission Realm
         Field permissionConfigField = pm.getClass().getDeclaredField("permissionRealmError");
         permissionConfigField.setAccessible(true);
-        final ObjectServerError error = new ObjectServerError(ErrorCode.UNKNOWN, "Boom");
+        final ObjectServerError error = new ObjectServerError(ErrorCode.WRONG_PROTOCOL_VERSION, "Boom");
```

Comment:
"Boom"?? What does that mean :-)

Reply:
Same for all the other "Boom"'s - is that propagated to the user? Then change to a better description.